src/platform/license-plan/license.plan.service.ts

```diff
@@ -23,6 +23,19 @@ export class LicensePlanService {
   ): Promise<ILicensePlan> {
     const licensePlan: ILicensePlan = LicensePlan.create();
     licensePlan.name = licensePlanData.name;
+    licensePlan.assignToNewOrganizationAccounts =
+      licensePlanData.assignToNewOrganizationAccounts;
+    licensePlan.assignToNewUserAccounts =
+      licensePlanData.assignToNewUserAccounts;
+    licensePlan.enabled = licensePlanData.enabled;
+    licensePlan.isFree = licensePlanData.isFree;
+    licensePlan.licenseCredential = licensePlanData.licenseCredential;
```

Comment:
the data is missing on `licensePlanData` in the first place, how is that fixing the issue?